huaweicloud/services/ccm/resource_huaweicloud_ccm_private_ca.go

```diff
@@ -423,18 +423,15 @@ func buildPrivateCARequestBodyKeyUsages(caType interface{}, rawParams interface{
 	return rawArray
 }
 
-func buildPrivateCARequestBodyCrlConfiguration(issuerID interface{}, rawParams interface{}) map[string]interface{} {
+func buildPrivateCARequestBodyCrlConfiguration(rawParams interface{}) map[string]interface{} {
 	if rawArray, ok := rawParams.([]interface{}); ok {
 		if len(rawArray) == 0 {
 			return nil
 		}
 		raw := rawArray[0].(map[string]interface{})
-		name := utils.ValueIngoreEmpty(raw["crl_name"]).(string)
-		if name == "" {
-			name = issuerID.(string)
-		}
```

Comment:
why we remove these codes?
there is no comment in your PR description: **why we need this PR**/**what was PR does**.

Reply:
done